apps/sms/views/inbox/js/inbox.js

```diff
@@ -124,6 +125,16 @@ var InboxView = {
       groupThreadTitleMaxLength: (window.innerWidth - 100) / 10
     });
 
+    this.once('fully-loaded', () => {
```

Comment:
I felt like these actions ```ensureReadAheadSetting``` and ```initStickyHeader``` are more related to the view itself rather then ```renderThreads``` methods - so moved them here. What do you think?